src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,480 @@
+    ```sql
+    SELECT average(duration) 
+    FROM Transaction 
+    SINCE 1 day ago 
+    TIMESERIES 1 hour
+    ```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 day ago using 1 hour buckets"
+  alt="A screenshot of a query selecting the average duration since 1 day ago using 1 hour buckets"
+  src={queriesnrqltutorial13}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    Notice this may return a flatter graph, since we only have 24 data points across the 1-day period being plotted. But what if you want to see the maximum possible granularity? Any timeseries query can have up to 366 data buckets, meaning the maximum granularity possible for 24 hours is to bucket our data into 4 minute windows. We could see this by using `TIMESERIES 4 minutes`, or we could ask NRDB to calculate this for us using `TIMESERIES MAX`:
+    ```sql
+    SELECT average(duration) 
+    FROM Transaction 
+    SINCE 1 day ago 
+    TIMESERIES MAX
+    ```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 day ago using the maximum number of buckets"
+  alt="A screenshot of a query selecting the average duration since 1 ago using the maximum number of buckets"
+  src={queriesnrqltutorial14}
+/>
+  </Side>
+</SideBySide>
+Now we're achieving new levels of visualization, breaking out from summarized numbers to line charts of data trends over time. As you can see, using `TIMESERIES` offers full control over visualizations, granularity, and averaging data over specified windows. Let's go to the next step: using the 'Where' clause.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="where"
+    title="5. Use Where clauses"
+  >
+<SideBySide>
+  <Side>
+    You may find that sometimes, you want your query to operate on a specific subset of NRDB data. For example, let's query our backend application's average response time. This data includes an attribute called `transactionType`, which specifies whether the transaction we timed was a "Web" or "Non-Web" (such as a cron job or queued task) transaction. Let's filter our query to only return data for the "Web" transactionType:
+    ```sql
+    SELECT average(duration) 
+    FROM Transaction 
+    WHERE transactionType='Web' 
+    TIMESERIES
+    ```
+  </Side>
+  <Side>
+    <img
+  title="Select the avarege duration for Web transaction types with a timeseries"
+  alt="A screenshot of a query selecting the average duration for Web transaction types with a timeseries"
+  src={queriesnrqltutorial15}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    You can also combine `AND` and `OR` to make more complex queries, using parentheses "(" and ")" to control how these work together.This query shows the average response time of "Web" transactions that were both faster than 100ms, and had a response code of either 200 or 302. Notice how we use parentheses to control the `OR`, which may behave differently than you'd expect.
```

Comment:
@jeff-colucci Looks like you need a space before "This query shows..."

Also, could you explain the AND/OR scenario you thought I might expect that isn't true with NRQL?

Reply:
Thank you! I looked at this more and I think that the information is a little vague as it is. I have made changes.